tests/Parser/parse.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+require_once dirname( __FILE__ ) . '/../../IABot/Parser/parse.php';
```

Comment:
You need to include the extension file as well.  Otherwise this thing will fatal error on you.